### PR TITLE
Add fx dependency to katello

### PR DESCRIPTION
**Pull request by ianballou**
Adds fx dependency to Katello for Katello Applicability.

### CODE REVIEW

**Comment by jlsherrill** [comment]
release needs a bump

**Comment by jlsherrill** [comment]
The reason this is failing is because the fx rpm is in the wrong comps.  It looks like it was put in comps/comps-foreman-rhel7.xml  but it needs to go in comps/comps-katello-server-rhel7.xml

sorry for not catching that earlier!

**Comment by jlsherrill** [comment]
and i would fix it in a 2nd pr

**Comment by ianballou** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
[test rpm]

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -57,6 +57,7 @@ Requires: %{?scl_prefix}rubygem(deface) >= 1.0.2
 Requires: %{?scl_prefix}rubygem(deface) < 2.0.0
 Requires: %{?scl_prefix}rubygem(angular-rails-templates) >= 1.0.2
 Requires: %{?scl_prefix}rubygem(angular-rails-templates) < 1.1
+Requires: %{?scl_prefix}rubygem(fx) < 1.0
```
Comment (ekohl) [inline packages/katello/rubygem-katello/rubygem-katello.spec:60]: This is not preset in the gemspec so the next automated update will remove this: https://github.com/theforeman/foreman-packaging/pull/5062

**Comment by ianballou** [comment]
Alright, I guess then we'll need to add it only after the Katello PR lands that requires Fx.

**Pull request merged**